ui/svelte.config.js

```diff
@@ -1,4 +1,4 @@
-import adapter from '@sveltejs/adapter-auto';
```

Comment:
if we remove the referenc to adapter-auto we can remove it in the package.json and lock

Reply:
ok